integration-tests/go.mod

```diff
@@ -56,6 +60,8 @@ require (
 	github.com/Masterminds/sprig/v3 v3.2.3 // indirect
 	github.com/Microsoft/go-winio v0.6.1 // indirect
 	github.com/Microsoft/hcsshim v0.11.4 // indirect
+	github.com/NethermindEth/juno v0.3.1 // indirect
+	github.com/NethermindEth/starknet.go v0.6.1-0.20231218140327-915109ab5bc1 // indirect
```

Comment:
probably unavoidable at the moment - but now we have starknet deps in solana integration tests 😢

Reply:
:shrug: 
Something to do with core integration-tests/ importing the chain family repos? I'm not sure.

Reply:
yeah it is - not much we can do about it at the moment tho